support-workers/src/test/scala/com/gu/support/workers/lambdas/UpdateSupporterProductDataSpec.scala

```diff
@@ -212,8 +203,12 @@ object UpdateSupporterProductDataSpec {
       "productType": "DigitalSubscriptionGiftPurchase"
     }
   """
-
-  lazy val catalog: String = Source.fromURL(getClass.getResource("/catalog.json")).mkString
+  lazy val catalogSource = Source.fromURL(getClass.getResource("/catalog.json"))
+  lazy val catalog: String = {
+    val catalogString = catalogSource.mkString
+    catalogSource.close()
+    catalogString
+  }
```

Comment:
I added this as I noticed the source wasn't closed (thanks IntelliJ). We close it once we've read the value. It has to be this was as they're lazy vals.